server.js

```diff
@@ -15,3 +15,5 @@ require('./models/init')().then(function () {
 }).catch(function(err) {
   console.log(err);
 });
+
+require('./adapters/software');
```

Comment:
I'm not sure we need this require() here. IMO, since you've registered your route file in the main route file, everything is require before hand (controller, manager AND adapter).